README.md

```diff
@@ -290,6 +290,44 @@ RPC options:
 ```
 <!-- END_CLI_HELP -->
 
+### Syncing with Holesky
+
+#### Step 1: Set up a jwt secret for both clients
+
+As an example, we put the secret in a `secrets` directory in the home folder.
+
+```bash
+sudo mkdir -p ~/secrets
+openssl rand -hex 32 | tr -d "\n" | sudo tee ~/secrets/jwt.hex
+```
+
+We will pass this new file’s path as an argument for both clients.
+
+#### Step 2: Launch Ethrex
+
+Pass holesky as a network and the jwt secret we set in the previous step.
+This will launch the node in full sync mode, in order to test out snap sync you can add the flag `--syncmode snap`.
+
+```bash
+cargo run --release --bin ethrex -- --http.addr 0.0.0.0 --network holesky --authrpc.jwtsecret ~/secrets/jwt.hex
+```
+
+#### Step 3: Set up a Consensus Node
+
+For this quick tutorial we will be using lighthouse, but you can learn how to install and run any consensus node by reading their documentation.
+
+You can choose your preferred installation method from [lighthouse's installation guide](https://lighthouse-book.sigmaprime.io/installation.html) and then run the following command to launch the node and sync it from a public endpoint
```

Comment:
Can we add an example users can run to install lighthouse on ubuntu/debian? (that's the more common servers people will run this on, at least for now). I imagine it can just be a `curl` command to download the binary and then a `mv` one to put it under the `PATH`.

Also, let's add which lighthouse version we use

Reply:
I prefer to link the installation page as it is very straightforward. We also use the latest version available, so I would rather omit it so we don't forget to update it every time there is a new lighthouse release